liquidator/src/liquidate.rs

```diff
@@ -572,18 +572,26 @@ impl<'a> LiquidateHelper<'a> {
         //
         // TODO: All these close ix could be in one transaction.
         if let Some(txsig) = self.perp_close_orders().await? {
-            return Ok(txsig);
+            return Ok(Some(txsig));
         }
         if let Some(txsig) = self.serum3_close_orders().await? {
```

Comment:
this cancels orders on a random market no? do you want to add todo to cancel all in future before moving on, or is that by design?

Reply:
It cancels a random one each time. Next time it'll cancel the next. There's a TODO to just cancel all in one tx in there somewhere.